docs/spec.md

```diff
@@ -2464,6 +2464,8 @@ Performs element-wise exponential operation on `operand` tensor and produces a
 // %result: (-1.13120438, 2.47172667)
```

Comment:
I think we should have one test per op spec. Make sure that is in sync with the Example in the ODS.

Reply:
Now Spec and tests are aligned for the 2 tests. ODS now has one example and is aligned with one example of them.